helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -387,12 +396,21 @@ spec:
           name: {{ template "splunk-otel-collector.fullname" . }}-fluentd-json
       {{- end}}
       {{- if eq .Values.logsEngine "otel" }}
+      {{- if .Values.isWindows }}
+      - name: varlog
+        hostPath:
+          path: C:/var/log
+      - name: varlibdockercontainers
```

Comment:
The name is misleading. Should be `programdata` perhaps.

Reply:
Thanks! renamed volume name with `programdata`